molecule/default/create.yml

```diff
@@ -38,6 +38,14 @@
         from_port: "{{ default_ssh_port }}"
         to_port: "{{ default_ssh_port }}"
         cidr_ip: "0.0.0.0/0"
+      - proto: tcp
+        from_port: 443
+        to_port: 443
+        cidr_ip: "0.0.0.0/0"
+      - proto: tcp
+        from_port: "0"
+        to_port: "65535"
+        group_name: "{{ default_security_group_name }}"
```

Comment:
moved back to the default security group in the create playbook because I didn't find an easy way to manage more than one security group on the nginx instance (tbh didn't want to spend more time on that one)